qcfractal/interface/models/common_models.py

```diff
@@ -196,3 +196,14 @@ def __init__(self, **data):
 
     def get_hash_index(self):
         return hash_dictionary(self.values.copy())
+
+
+class Citation(ProtoModel):
+    """ A literature citation.  """
+    _citation: str  # hand-formatted citation in ACS style. In the future, this could be bibtex, rendered to different formats.
+    doi: Optional[str]
+    url: Optional[str]
```

Comment:
This should optionally allow full definitions:
```
author: List[str]
title: str
journal: str
year: int
volume: int
number: int
pages Tuple[int, Optional[int]]
month: int
doi: str
note: str
key: str
```

Reply:
The plan is to store the citation object as a JSON blob in the DB to allow for future improvement. At this stage, we will pre-render citations in `_citation`, but later we will have the option of using the citation manager tech to render citations from bibtex.

Reply:
Sounds good, but lets make sure we have the option and correct places to keep all of the required metadata currently. Going back and having to re-enter the information seems rough.